### PR TITLE
Update WordprocessingDocumentFactory to handle Numbering definitions

### PR DESCRIPTION
**Pull request by kylegibson**
Also update the DocumentGeneratorTestCase to handle additional explicit arbitrary parts

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -36,7 +36,7 @@ docs:
 	$(MAKE) -C docs html
 
 release: clean
-	python setup.py sdist bdist_wheel upload
```
Comment (jlward) [inline Makefile]: I don't even know what `bdist_wheel` does.

Comment (kylegibson) [inline Makefile]: It came with the `Makefile` template I used. It doesn't work, so I removed it for now. We can always add it back later if for some reason it was actually useful.

File: tests/export/test_html.py [hunk 2]
```diff
@@ -1678,3 +1817,41 @@ def test_with_line_break(self):
 
         expected_html = '<p><a href="http://google.com">li<br />nk</a>.</p>'
         self.assert_document_generates_html(document, expected_html)
+
+
+class NumberingTestCase(DocumentGeneratorTestCase):
+    def test_lowerLetter_numbering_format_is_handled(self):
+        num_id = '2'
+        numbering_xml = '''
+            <num numId="{num_id}">
```
Comment (jlward) [inline tests/export/test_html.py]: I assume this is was the thing you showed me the other day?

**Comment by jlward** [comment]
Looks good.

File: tests/export/test_html.py (hunk 2 above)
Comment (kylegibson) [inline tests/export/test_html.py]: No, it's not. I added support for handling numbering definitions to the DocumentGeneratorTestCase. It didn't make sense to do that without having at least one test case in the pydocx repo that used it, so I added one.

Comment (jlward) [inline tests/export/test_html.py]: Makes sense.

**Pull request merged**